web/packages/teleport/src/Welcome/NewCredentials/NewCredentials.tsx

```diff
@@ -66,6 +66,9 @@ export function NewCredentials(props: NewCredentialsProps) {
     displayOnboardingQuestionnaire = false,
     setDisplayOnboardingQuestionnaire = false,
     Questionnaire = undefined,
+    displayInviteCollaborators = false,
+    setDisplayInviteCollaborators = false,
```

Comment:
i'd avoid naming things `setXXX` unless it's a react hook state setter (that's how it's normally named `setXXX`), not sure how to give rename suggestions b/c i don't really understand it yet

Reply:
Right, sorry, I was following the pattern from `setDisplayOnboardingQuestionnaire` but I think I can just as well set it to `null`.